realm/realm-library/src/objectServer/java/io/realm/mongodb/mongo/iterable/MongoCursor.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.mongodb.mongo.iterable;
+
+import com.google.android.gms.tasks.Task;
+
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+
+/**
+ * The Mongo Cursor class.
```

Comment:
This doesn't say anything the name of the class doesn't. Maybe expand on what the class actually is for?